dino_webpage/styles.css

```
@@ -1,148 +1,32 @@
-}
-
-.nav__logo{
-    width: 80%;
-    margin-top: 6px;
-    margin-left: 1%;
-    /* height:10px; */
+.magic-bento-card::after {
+    --glow-x: 50%;
+    --glow-y: 50%;
+    --glow-color: var(--primary);
+    opacity: 0;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)